openapi/adintel/adintel.yaml

```diff
@@ -192,6 +287,47 @@ components:
         status: trustedTester
       x-tags:
         - accountStats
+    connectedAccount:
+      title: connectedAccount
+      type: object
+      description: 'An advertising account that’s connected within Advertising Intelligence from integrated campaign management platforms, including Google Adwords, Facebook Ads and Local Ads.'
+      x-examples: {}
+      properties:
+        id:
+          type: string
+          description: The resource ID of the Advertising Account
+        type:
+          type: string
+          description: connectedAccount
+        attributes:
+          type:
+            - object
+          properties:
+            name:
+              type: string
+              description: The name of the ad account
+            currencyCode:
+              type: string
+              description: "\t\nThe currency code that the ad account uses"
+            credentialInvalid:
```

Comment:
should be `credentialsInvalid`

Reply:
When reviewing @wfehr-va's endpoint we decided that this should be a `status` enum. In the future, we may add a `statusReason` field to indicate why the credentials are invalid. 

Lets try to keep the enums the same with https://developers.vendasta.com/platform/branches/products/c2NoOjI4NDA1NDcw-social-profiles